CONTRIBUTING.rst

```diff
@@ -40,3 +44,59 @@ To bring in updated upstream modules.  Update `gilt.yml` and execute the followi
 
 .. _`Ansible Goss`: https://github.com/indusbox/goss-ansible
 .. _`Gilt`: http://gilt.readthedocs.io
+
+
+Move to Red Hat
+===============
+
+During the end of October 2018 the Molecule was moved to its new home under Ansible by Red Hat.
+
+Update Git repo location
+------------------------
+
+The Molecule project has moved:
+
+old location: ``https://github.com/metacloud/molecule``
+
+new location: ``https://github.com/ansible/molecule``
+
+If you have the source checked out you should use ``git remote set-url origin``
+to point to the new location
```

Comment:
Add a . to the end of the sentence.

Reply:
@acozine try out new GitHub Suggestions feature. Just hit Ctrl+G in the comment box and change the line as you want. It'll appear like:
```suggestion
to point to the new location.
```